src/coreclr/tools/aot/ObjWriter/llvm.patch

```diff
@@ -454,26 +153,11 @@ index 31762b9e4cd50e429725a3f32323cc5a555dfe2f..d53ab3e3b61c2fd73e7333682289d310
    case FK_SecRel_2:
      return COFF::IMAGE_REL_ARM64_SECTION;
  
-diff --git a/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp b/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp
-index a77df7a2598f4176ccfda98c72c35471a679285b..e1aa7526f9b07205ca27ab0489e11a6081bcf8e9 100644
---- a/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp
-+++ b/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp
-@@ -48,6 +48,14 @@ public:
- };
- } // end anonymous namespace
- 
-+Optional<MCFixupKind> ARMAsmBackend::getFixupKind(StringRef Name) const {
```

Comment:
Should we keep this as well instead of the change in `ObjectWriter::EmitSymbolRef`?